lib/utils/telemetry.js

```diff
@@ -36,12 +36,18 @@ function getTelemetry() {
  */
 function getTelemetryFor(filePath) {
   let modulePath = getModulePathFor(filePath);
-  let moduleKey = modulePath.replace('templates/components/', 'components/');
+  let moduleKey = _generateModuleKey(modulePath);
   let data = getTelemetry()[moduleKey];
 
   return data;
 }
 
+function _generateModuleKey(modulePath) {
+  let moduleKey = modulePath.replace('templates/components/', 'components/');
+  // If `templates/` still exists in the path then it wasn't a component but a controller-level template instead
+  return moduleKey.replace('templates/', 'controllers/');
```

Comment:
I'm unsure if this change will have other implications for other app types, i.e. pods, MU, but from my understanding it doesn't work with pods yet anyway 🤔

Reply:
:+1: We'll get there when we get there :)